ddtrace/tracer/option.go

```diff
@@ -654,11 +658,13 @@ func WithDogstatsdAddress(addr string) StartOption {
 	}
 }
 
-// WithSamplingRules specifies the sampling rates to apply to spans based on the
+// WithSamplingRules specifies the sampling rates to apply to trace spans based on the
 // provided rules.
+// todo(shevchenko): there might be a need to add an analogous option for
```

Comment:
```suggestion
// TODO(shevchenko): there might be a need to add an analogous option for
```
I don't think the `TODO` note should be part of public documentation. Maybe move it below the function signature.